.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+InlineHtmlStripStylesTransformer.js
```

Comment:
would be cool to convert the source code to TS some day and have a `build/` directory where we could point the package. not in scope of this PR of course.

Reply:
So I keep the generated `InlineHtmlStripStylesTransformer.js` in `.gitignore` or do you suggest I should commit it as well?
The js-files could be easily moved to the `src`-folder, tsc will just keep them compiled. But would add a lot of noise to this PR, which is quite big already. I would prefer to do it another time.